src/main/java/frc/robot/controls/ManualController.java

```diff
@@ -28,5 +35,16 @@ public void configureControls() {
     m_manual.get(Button.Y).onTrue(new InstantCommand(() -> m_arm.setMotorPower(0.05)));
     m_manual.get(Button.X).onTrue(new InstantCommand(() -> m_arm.setMotorPower(-0.05)));
     m_manual.get(Button.B).onTrue(new InstantCommand(() -> m_arm.setMotorPower(0)));
+
+    //Move to min height
+    m_manual.get(m_manual.RIGHT_TRIGGER_BUTTON).onTrue(new MoveToHeight(m_elevator, ElevatorConstants.kMinHeight)); 
+    //Calibrate elevator using inbuilt motor encoders
+    m_manual.get(DPad.DOWN).onTrue(new ResetEncoderAtBottom(m_elevator));
+    //TODO: calibrate elevator using absolute encoders(probably will not work yet as of 2/15/2023);
+    //move to bottom node height
+  
+   
+
+  
```

Comment:
Remove random spaces, I know it is knit picky but it is annoying for a few reasons. It make the file change view larger than it needs to be. It makes random files appear in the pull request. Random spaces can add up and It makes random gaps of code which increases the distance to scroll though code. Everyone dose it but at least clean it up before you make a PR. So I know it's knit picky but it makes is easier for the people reviewing and working on the code.